From 346dd44466700cff3fb85ef8e4b8274a9acf522c Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 2 Apr 2015 12:44:23 +0200
Subject: [PATCH 47/48] plugins: use cached copy of plugin description list at
 iridiumbrowsing.de

The list contains mimetype-to-plugin mappins, as well as blacklists
for security-impeded old versions of plugins.
---
 chrome/browser/plugins/plugins_resource_service.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/plugins/plugins_resource_service.cc b/chrome/browser/plugins/plugins_resource_service.cc
index d43526e..c3e514a 100644
--- a/chrome/browser/plugins/plugins_resource_service.cc
+++ b/chrome/browser/plugins/plugins_resource_service.cc
@@ -21,7 +21,7 @@ const int kStartResourceFetchDelayMs = 60 * 1000;
 const int kCacheUpdateDelayMs = 24 * 60 * 60 * 1000;
 
 const char kPluginsServerUrl[] =
-    "https://www.gstatic.com/chrome/config/plugins_2/";
+    "https://safebrowsing.iridiumbrowser.de/";
 
 GURL GetPluginsServerURL() {
   std::string filename;
-- 
2.4.0

